sklearn/preprocessing/label.py

```diff
@@ -90,6 +91,12 @@ def _encode(values, uniques=None, encode=False):
         already have been determined in fit).
     encode : bool, default False
         If True, also encode the values into integer codes based on `uniques`.
+    check_unknown : bool, default True
+        If True, check for values in ``values`` that are not in ``unique``
+        and raise an error. This is ignored for object dtype and treated as
+        True in this case). This parameter is useful for
+        _BaseEncoder._transform() to avoid calling _encode_check_unknown()
+        twice.
```

Comment:
Maybe move comment about why this is useful to before the `_encode` call in `_BaseEncoder._transform` so that a reader of `_BaseEncoder._transform` knows why `check_unknown=False`.